src/monitor/monitor.c

```diff
@@ -2240,6 +2240,10 @@ static void service_startup_handler(struct tevent_context *ev,
     DEBUG(SSSDBG_FATAL_FAILURE,
           "Could not exec %s, reason: %s\n", mt_svc->command, strerror(errno));
 
+    /* Also log to syslog since SSSD failed to start. */
+    sss_log(SSS_LOG_ERR, "Could not exec %s, reason: %s",
```

Comment:
SSSDBG_FATAL_FAILURE is the most critical type of error for DEBUG().
For the sss_log() the most critical is SSS_LOG_EMERG.
Here we are using SSS_LOG_ERR which maps more to SSSDBG_MINOR_FAILURE than SSSDBG_FATAL_FAILURE.
If this should be unified this way or DEBUG() log level do not need to be strictly correlated with sss_log() level?

Reply:
I'm actually unsure about the level. EMERG should be used when the whole system is unusable. ALERT seems to be too much to me as well and both of these will result in broadcasting the message to console. So I'd choose only ERROR or CRIT, both seems to be fine to me.

Reply:
Hmmm, those log level adjustments may be material for another PR. So for current one LGTM.